### PR TITLE
Added notebook for the Simplified mode in POT

### PR DESCRIPTION
**Pull request by Noktyrn**
Ticket: [CVS-78250]

### CODE REVIEW

**Comment by MaximProshin** [comment]
Do we really need to have the images inside the notebook?

**Comment by Noktyrn** [comment]
> Do we really need to have the images inside the notebook?

In the simplified mode we need to provide a path to the folder with images from the dataset. I've noticed that some other notebooks do also have such folder, and that's why I thought that it's not a problem.
Would it be better, if I downloaded and extracted the images into the data folder instead?

**Comment by MaximProshin** [comment]
@helena-intel , @ryanloney , is it ok to have the folder with images in an OV notebook?

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 1]
```diff
@@ -0,0 +1,179 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "POT Simplified Mode tutorial"
```
Comment (MaximProshin) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb:7]: I propose adding some short intro about the Simplified mode which works with CV only and doesn't guarantee an acceptable accuracy drop.

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+   "outputs": [],
+   "source": [
+    "!pot -c config.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the next releases the following method for using the simplified mode quantization without writing a POT config will be available"
```
Comment (MaximProshin) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb:131]: I don't like it. I propose to align it with 2022.1

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.5"
+  }
```
Comment (MaximProshin) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb:175]: something weird

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pot -q default -m $ir_model_xml -w $ir_model_bin --engine=simplified --data-source=$DATA_DIR"
+   ]
+  }
+ ],
```
Comment (MaximProshin) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb:143]: I believe we should add performance measurements at least. I'm not sure about the accuracy as on the one hand we want to avoid AC and on the other hand it would be nice to show the accuracy.

**Review by MaximProshin — CHANGES_REQUESTED** [review]

File: notebooks/114-simplified-mode/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Accelerate Inference of VGG19 Image Classification Model using the Simplified Mode of OpenVINO Post-Training Optimization Tool 
+
+This tutorial demostrates how to apply INT8 quantization to the
+Image classification model VGG19, using the [Post-Training Optimization
+Tool
+API](https://docs.openvinotoolkit.org/latest/pot_compression_api_README.html)
```
Comment (MaximProshin) [inline notebooks/114-simplified-mode/README.md:6]: It should be corrected POT API => POT Simplified Mode

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 6]
```diff
@@ -0,0 +1,585 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# POT Simplified Mode tutorial\n",
```
Comment (MaximProshin) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb:7]: Let's improve it: INT8 Quantization by POT in Simplified Mode tutorial

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb (hunk 3 above)
Comment (MaximProshin) [reply notebooks/114-simplified-mode/114-simplified-mode.ipynb:581]: it's still valid

**Review by MaximProshin — APPROVED** [review]

**Comment by MaximProshin** [comment]
@helena-intel , @ryanloney , @paularamo , can someone from your side also review this notebook?

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# INT8 Quantization by POT in Simplified Mode tutorial\n",
+    "\n",
+    "This tutorial shows how to quantize a ResNet20 image classification model, trained on CIFAR10 dataset, using the Simplified Mode of OpenVINO's Post-Training Optimization Tool. \n",
+    "\n",
+    "With the Simplified Mode models can be quantized without writing an accuracy checker configuration file. The user just need to specify the folder with images, which will later be used as calibration dataset for POT.\n",
```
Comment (AlexKoff88) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb:11]: I don't think it is a good idea to mention accuracy checker without even providing links to it. I would remove this mention at all and revise the information about Simplified mode. You can take it from here: https://github.com/openvinotoolkit/openvino/blob/master/tools/pot/docs/SimplifiedMode.md

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 8]
```diff
@@ -0,0 +1,367 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config = dict({\n",
+    "    \"model\": {\n",
```
Comment (AlexKoff88) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb:150]: Since there is nothing custom in the config for this model. Can we remove it and use POT purely from CLI so that the POT config will be generated on-fly. We started supporting this in 2022.1.

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 9]
```diff
@@ -0,0 +1,367 @@
...
+   "source": [
+    "# Inference INT8 model (IR)\n",
+    "!benchmark_app -m $optimized_model_xml -d CPU -api async"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Demonstration of the results"
```
Comment (AlexKoff88) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb:241]: I would anything about what `Demonstration of the results` is about what users should expect to see.

**Comment by ryanloney** [comment]
@Debskij @adrianboguszewski can you please help review this now that we have 2022.1 in main

**Review by AlexKoff88 — APPROVED** [review]

**Comment by AlexKoff88** [comment]
@Noktyrn, please rebase your branch.

Any chance we can merge this at the nearest time?

**Review by ryanloney — CHANGES_REQUESTED** [review]
Please address the issue reported by @adrianboguszewski. This needs to pass CI

File: notebooks/114-simplified-mode/114-simplified-mode.ipynb [hunk 10]
```diff
@@ -0,0 +1,350 @@
...
+    "In the first step the network is loaded using the IECore:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ie = IECore()\n",
```
Comment (AlexKoff88) [inline notebooks/114-simplified-mode/114-simplified-mode.ipynb]: @Noktyrn, please revise this code using the new OpenVINO API. Here is a guide how to use it: https://docs.openvino.ai/latest/openvino_docs_Integrate_OV_with_your_application.html

**Comment by Noktyrn** [comment]
I updated the notebook, now it doesn't use the old API and has the correct name

File: notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+    "import warnings\n",
+    "\n",
+    "import torch\n",
+    "from torchvision import transforms as T\n",
+    "from torchvision.datasets import CIFAR10\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "import openvino.runtime as ov\n",
```
Comment (paularamo) [inline notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb:41]:
```suggestion
    "import openvino.runtime as Core\n",
```

File: notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+    "In the first step the network is loaded using the IECore:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ie = ov.Core()\n",
```
Comment (paularamo) [inline notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb:213]:
```suggestion
    "ie = Core()\n",
```

**Review by paularamo — CHANGES_REQUESTED** [review]

File: notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb [hunk 13]
```diff
@@ -0,0 +1,353 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "indexes_to_infer = [0, 1, 2]  # to plot specify indexes\n",
+    "\n",
+    "plot_pictures(indexes_to_infer)\n",
+    "\n",
+    "results_quanized = infer_on_pictures(compiled_model, indexes_to_infer)\n",
+    "\n",
+    "print(f\"Labels for picture from quantized model : {results_quanized}.\")"
```
Comment (ryanloney) [inline notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb:314]:
```suggestion
    "print(f\"Image labels using the quantized model : {results_quanized}.\")"
```

File: notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb [hunk 14]
```diff
@@ -0,0 +1,353 @@
...
+ "metadata": {
+  "accelerator": "GPU",
+  "colab": {
+   "collapsed_sections": [
+    "K5HPrY_d-7cV",
+    "E01dMaR2_AFL",
+    "qMnYsGo9_MA8",
+    "L0tH9KdwtHhV"
+   ],
+   "name": "INT8 Quantization by POT in Simplified Mode tutorial",
```
Comment (ryanloney) [inline notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb:327]:
```suggestion
   "name": "INT8 Quantization with POT in Simplified Mode tutorial",
```

File: notebooks/README.md [hunk 15]
```diff
@@ -55,7 +55,8 @@ More amazing notebooks here! Please take a look of the complete list.
 | [110-ct-segmentation-quantize](110-ct-segmentation-quantize/)<br> | Quantize a kidney segmentation model and show live inference | 
 | [111-detection-quantization](111-detection-quantization/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F111-detection-quantization%2F111-detection-quantization.ipynb) | Quantize an object detection model | 
 | [112-pytorch-post-training-quantization-nncf](112-pytorch-post-training-quantization-nncf/) | Use Neural Network Compression Framework (NNCF) to quantize PyTorch model in post-training mode (without model fine-tuning)| 
-| [113-image-classification-quantization](113-image-classification-quantization/) | Quantize mobilenet image classification | 
+| [113-image-classification-quantization](113-image-classification-quantization/) | Quantize mobilenet image classification |
```
Comment (ryanloney) [inline notebooks/README.md:58]:
```suggestion
| [113-image-classification-quantization](113-image-classification-quantization/) | Quantize Image Classification Models with POT |
```

File: notebooks/README.md [hunk 16]
```diff
@@ -55,7 +55,8 @@ More amazing notebooks here! Please take a look of the complete list.
...
+| [114-quantization-simplified-mode](114-quantization-simplified-mode/) | Quantize ResNet20 image classification using Simplified Mode| |
```
Comment (ryanloney) [inline notebooks/README.md:59]:
```suggestion
| [114-quantization-simplified-mode](114-quantization-simplified-mode/) | Quantize Image Classification Models with POT in Simplified Mode| |
```

**Review by ryanloney — CHANGES_REQUESTED** [review]
Please review the suggested changes

File: notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb [hunk 17]
```diff
@@ -0,0 +1,353 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ie = ov.Core()\n",
+    "\n",
+    "compiled_model = ie.compile_model(str(optimized_model_xml), \"AUTO\")\n",
+    "\"\"\"\n",
+    "# read and load quantized model\n",
+    "quantized_net = ie.read_network(\n",
```
[the remark this answers is not in the document]
Comment (Noktyrn) [reply notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb:218]: Sorry, forgot to erase the old code, I will fix it as soon as possible.

**Review by adrianboguszewski — APPROVED** [review]

**Comment by adrianboguszewski** [comment]
I'm ok to merge it if you don't have more comments @ryanloney

File: notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb [hunk 18]
```diff
@@ -0,0 +1,346 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ie = Core()\n",
+    "\n",
+    "compiled_model = ie.compile_model(str(optimized_model_xml), \"AUTO\")"
+   ]
```
Comment (paularamo) [inline notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb:216]: @Noktyrn After running this, I got this error.

image

Comment (Noktyrn) [reply notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb:216]: Sorry @paularamo, but I couldn't reproduce this issue. Can you please tell me, if the IR was created in the compressed\optimized folder?

**Review by paularamo — APPROVED** [review]
